### DISCUSSION TITLE
Propriedade visible da PoPageDynamicTableCustomAction pode ser ativada por linha?

### DISCUSSION THREAD

**Discussion by gtrabach** [po-ui/po-angular#1766] [Tire suas dúvidas] [answered]
Ao utilizar uma Dynamic Table (PoPageDynamicTableComponent), desejo disponibilizar algumas opções para o usuário na tabela. Porém algumas opções serão habilitadas ou não de acordo com o status do registro.

Estou realizando testes com as propriedades da _PoPageDynamicTableCustomAction_.

Utilizando a propriedade **disabled** para desabilitar o botão já me atenderia, mas, gostaria que o mesmo sequer fosse exibido nessas situações.

Fiz alguns testes com a propriedade **visible**, mas ou ela está exibindo o botão em todas as linhas ou em nenhuma. Também não achei nenhum exemplo na documentação com essa propriedade.

Saberiam me informar se ela pode ser usada para habilitar botões por linhas? Exemplo: tenho três registros na grid, um com status 1 e demais com o status 2, preciso que o botão seja exibido no registro com status 1 e que seja ocultado nos registros com status 2.

Caso essa propriedade não permitir, existiria alguma outra?

Grato.

**Comment by anliben** [comment]
Boa Tarde @gtrabach, tudo bem?

Recebemos sua issue e em breve iremos analisar e retornar uma resposta, inclusive fique a vontade caso queria abrir PR com as mudanças propostas.

Attr.

**Comment by jhosefmarks** [comment, marked as answer]
@gtrabach dá uma olhada no sample [PO Page Dynamic Table - Users](https://po-ui.io/documentation/po-page-dynamic-table?view=web), o mesmo tem o comportamento que você deseja.
